protocol/contracts/templegold/DaiGoldAuction.sol

```diff
@@ -0,0 +1,262 @@
+pragma solidity 0.8.20;
+// SPDX-License-Identifier: AGPL-3.0-or-later
+// Temple (templegold/DaiGoldAuction.sol)
+
+
+import { CommonEventsAndErrors } from "contracts/common/CommonEventsAndErrors.sol";
+import { TempleElevatedAccess } from "contracts/v2/access/TempleElevatedAccess.sol";
+import { IDaiGoldAuction } from "contracts/interfaces/templegold/IDaiGoldAuction.sol";
+import { ITempleGold } from "contracts/interfaces/templegold/ITempleGold.sol";
+import { AuctionBase } from "contracts/templegold/AuctionBase.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { TempleMath } from "contracts/common/TempleMath.sol";
+
+/** 
+ * @title AuctionEscrow
+ * @notice Bidding token is deposited into this contract to bid on a share of distributed Temple Gold for an epoch.
+ *         Temple Gold acquired in past epochs can always be claimed. Once bidding for an epoch has ended, users cannot
+ *         claim their bid token and can claim their share of Temple Gold for epoch.
+ *         Elevated access can change bidding token for future epochs.
+ */
+contract DaiGoldAuction is IDaiGoldAuction, AuctionBase, TempleElevatedAccess {
+    using SafeERC20 for ITempleGold;
+    using SafeERC20 for IERC20;
+    using TempleMath for uint256;
+    
+    /// @notice Temple GOLD address
+    ITempleGold public immutable override templeGold;
```

Comment:
You can remove override as it does not override anything
You can remove all override in this contract, I have compiled successfully after I've removed them all

Reply:
Yes, it will compile successfully. 
We use override here for all public variables and functions defined in the interface as a standard. It also makes it easier to not forget implementing a derived interface function or vice versa.